DiamondLightSource/python-artemis
# 716 full grid scan sets transmission first

Fixes #716 and #744 

This PR uses code in #742 , so is waiting on this

### To test:
1. Confirm both requirements are met
2. Run tests
